files/chef-marketplace-gem/lib/marketplace/reckoner/eetee.rb

```diff
@@ -0,0 +1,86 @@
+require 'json'
+require 'net/http'
+require 'securerandom'
+require 'marketplace/reckoner/machine_id'
+
+class Marketplace
+  class Reckoner
+    class Eetee
+      include Marketplace::Reckoner::MachineID
+
+      attr_reader :metrics_to_collect, :run_period
+      attr_accessor :payload
+
+      VERSION = '1'.freeze
+
+      def self.daily_runner
+        metrics_to_collect = Marketplace::Reckoner::Config.phone_home.metrics.daily
+        return if metrics_to_collect.nil? || metrics_to_collect.empty?
+
+        runner = new(metrics_to_collect)
+        return if runner.disabled? || runner.endpoint.nil?
+
+        runner.run
+      end
+
+      def self.disabled_in_config?
```

Comment:
Does it make sense to change the class methods to `enabled?` and `disabled?` since it shadows the actual values we're checking?